ray_on_golem/client/client.py

```diff
@@ -19,6 +25,9 @@ def __init__(self, base_url: URL) -> None:
 
         self._session = requests.Session()
 
+        if not is_running_on_golem_network():
+            self._start_webserver()
```

Comment:
wondering aloud... is it sensible to put such long-blocking operations into a constructor? my intuition suggests this should be a separate step...

plus, it sounds somewhat unintuitive in general to start a _server_ when you instiantiate a _client_ ...

Reply:
Well... It sounded better at first, but good point. Moving this code to node provider fixes the problem of ray logs while starting webserver.